process-engine/skeleton
# Feature/docker image names

## What did you change?

I made the docker-images that are created during speedtesting not have possibly conflicting names.

Before this change, only the build-number was used to tag the image. this could've resulted in builds from different branches but same buildnumber overwriting each others docker images. This fixes that.

## How can others test the changes?

You *could* in theory create two branches here and commit on both of them at the same time and see how the tests actually reflect whatever is in the respective branch, and not in the other one.

## PR-Checklist

Please check the boxes in this list after submitting your PR:
